### DISCUSSION TITLE
go-sdk DeleteByIds transport is closing unexpectedly

### DISCUSSION THREAD

**Discussion by OsmanFarukO** [milvus-io/milvus#16139] [Q&A and General discussion]
Hi everyone,
I am using milvus 1.1.0 standalone on production and i want to insert data with using milvus/go-sdk. I am deleting data if exist and inserting data. When deleting data from milvus i got transport is closing error. I cant figure out why this happening and cant see log message from milvus. Can anyone help me with this problem.
Thank you.

![Screenshot from 2022-03-22 10-23-38](https://user-images.githubusercontent.com/24430724/159430780-aa5e228d-95bc-4493-bf5e-7c175fa98555.png)

**Comment by xiaofan-luan** [comment]
I thought milvus go sdk is only for 2.0 right?

**Reply by OsmanFarukO** [reply]
I am using github.com/milvus-io/milvus-sdk-go with version v1.1.0. I figure out why i am stuck on this error. Bad algorithm. Thanks for helping.
